docker-compose.yml

```diff
@@ -2,27 +2,37 @@ version: '3'
 
 services:
   djangoapp:
-    container_name: djangoapp
+    container_name: djangoapp-${ENVIRONMENT_NAME}
+    # find a way so we dont have to toggle this on the server
```

Comment:
What is the problem with using the environment variable to change the container name?

Reply:
That part is fine, but the lines below are the issue: we currently build locally and on GitHub using docker compose build or docker compose up --build, but one the server we use a pre-built image, so we can't use "build: ." there but instead something like "image: djangoapp:latest". The easiest solution is probably to put that second part (image:) into the compose file and use `docker build` to build instead of compose.

Reply:
Ah, yes. So compose does just that. Compose. Sounds good.